src/test/HostActivationTests/GivenThatICareAboutMultilevelSharedFxLookup.cs

```diff
@@ -595,7 +615,12 @@ public void Ensure_On_NonWindows_Multiple_Global_Locations_Are_Probed()
             {
                 return;
             }
-            
+            else
+            {
+                // Currently no support for global locations on Linux; remaining test code below is preserved for now.
```

Comment:
In general, I dislike this approach.  We have the test code preserved in source control history.  If we ever want to use it again, we can bring it back that way.

This just causes maintenance issues because as things get refactored, people will be forced to update this code to keep it building.  However, there is no way of knowing if the updates are valid or not because the test can't be executed.

IMO - we should just remove dead code, and use source code control for history purposes.